tests/fuzzer/afl-tests/literals.toit

```diff
@@ -0,0 +1,10 @@
+main:
```

Comment:
I assume these files cannot have the copyright header + license?

Reply:
The fuzzer would use it as input. So theoretically possible.
Will think about it when adding the copyright headers.